realm/realm-jni/src/io_realm_internal_table.cpp

```diff
@@ -1693,3 +1693,20 @@ JNIEXPORT jboolean JNICALL Java_io_realm_internal_Table_nativeHasSameSchema
 {
     return *TBL(thisTablePtr)->get_descriptor() == *TBL(otherTablePtr)->get_descriptor();
 }
+
+
+JNIEXPORT jlong JNICALL Java_io_realm_internal_Table_nativeVersion(
+        JNIEnv* env, jobject, jlong nativeTablePtr)
+{
+    bool valid = (nativeTablePtr != 0);
+    if (valid) {
+        if (!TBL(nativeTablePtr)->is_attached()) {
+            ThrowException(env, TableInvalid, "Table is closed, and no longer valid to operate on.");
+            return 0;
+        }
+    }
+    try {
```

Comment:
Hmm, this is the same impl in `Java_io_realm_internal_TableView_nativeSync` 

`Table#createNative` i.e `Java_io_realm_internal_Table_createNative` also return 0 as a fallback, hence the check for 0 for the `nativeTablePtr`